### PR TITLE
SMQ-209 - Fix Warnings for .svg Files During `npm run build`

### LINKED ISSUE #209: Bug: Warnings for SVG Files During `yarn build`
### What were you trying to achieve?

I was attempting to build the Docusaurus site using `yarn build`, ensuring that all SVG files render correctly on the generated website.

### What are the expected results?

The build process completes successfully without any warnings or errors, and all SVG images render correctly on the website.

### What are the received results?

While the build process completes successfully and the website renders correctly when served with `yarn start`, multiple warnings are displayed in the terminal for several SVG files. The warnings repeat for multiple files, as shown below:
```

[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer_3.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer_3.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_editor.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_editor.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_administrator.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_administrator.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_editor.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_editor.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_viewer.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_viewer.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_administrator_1.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_administrator_1.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_member.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_member.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_administrator_2.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_administrator_2.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_administrator_3.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_administrator_3.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer_2.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer_2.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer_1.drawio.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_viewer_1.drawio.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_12.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_12.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_13.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_13.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_11.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_11.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_14.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/group_users_member_14.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_administrator.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_administrator.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_editor.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_editor.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_viewer.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_viewer.svg)
[WARNING] The image at "/home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_member.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: /home/musilah/ULTRAVIOLET/docs/docs/diagrams/domain_users_member.svg)
```

### Steps To Reproduce

1. Run `npm install` once on the `main` branch of the repository.
2. Then run `yarn build` .
3. Run `yarn start` after to check the website and navigate to the authorization section of the docs.

### In what environment did you encounter the issue?

On the `main` branch of the repository

### Additional information you deem important

_No response_

### PR DESCRIPTION
**Pull request by Musilah**
### What does this do?

This fixes the warning that appears after a user runs `yarn build`. This warning has been cleared.

There is however a Babel warning that can be ignored since it does not really affect anything and the [Docusaurus](https://github.com/babel/babel/discussions/13676) team has advised to ignore as well. But to minimise the warning, I have compacted the other .svgs to below `500KBs` except for one which is still at `685KBs`.

### Which issue(s) does this PR fix/relate to?

Resolves #209 

### List any changes that modify/break current functionality

- [x] Fix svg files warnings
- [x] Add linter to workflows
- [x] Compact the .svg files that are affected by Babel

### Have you included tests for your changes?
No
### Did you document any new/modified functionality?
No

### CODE REVIEW

**Review by ianmuchyri — DISMISSED** [review]

**Review by dborovcanin — CHANGES_REQUESTED** [review]
image
Rename `Device` to `Client`.

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Also add Mermaid support https://docusaurus.io/docs/next/markdown-features/diagrams.

**Review by dborovcanin — DISMISSED** [review]

File: src/pages/landing/index_backup.tsx
Comment (ianmuchyri) [inline src/pages/landing/index_backup.tsx:1]: Since we no longer have a landing page, do we need this file? More so, do we need this landing folder?

Comment (ianmuchyri) [reply src/pages/landing/index_backup.tsx:1]: Also remove `open_source.svg` and `scalability.svg` images that is used in this page if we are to delete this page

**Review by ianmuchyri — CHANGES_REQUESTED** [review]

**Review by ianmuchyri — APPROVED** [review]

**Pull request merged**